tests/stubtest_third_party.py

```diff
@@ -153,6 +163,84 @@ def run_stubtest(
     return True
 
 
+def setup_gdb_stubtest_command(venv_dir: Path, stubtest_cmd: list[str]) -> bool:
+    """
+    Use wrapper scripts to run stubtest inside gdb.
+    The wrapper script is used to pass the arguments to the gdb script.
+    """
+    if sys.platform == "win32":
+        print_error("gdb is not supported on Windows")
+        return False
+
+    gdb_version = subprocess.check_output(["gdb", "--version"], text=True, stderr=subprocess.STDOUT)
```

Comment:
This line will fail with `FileNotFoundError` if gdb is not installed.

```suggestion
    try:
        gdb_version = subprocess.check_output(["gdb", "--version"], text=True, stderr=subprocess.STDOUT)
    except FileNotFoundError:
        print_error("gdb is not installed")
        return False
```